src/components/SubjectsPanel/EnrolledNowSection.tsx

```
@@ -6,11 +6,17 @@ import { SubjectRow } from './SubjectRow';
 import { isZameraniCode } from './utils';
 
 // Winter semester: Sep 1 – Jan 31. Summer semester: Feb 1 – Aug 31.
-function isThisSemester(fulfillmentDate?: string): boolean {
+// fulfillmentDate is DD.MM.YYYY in Czech (dot separator) but MM/DD/YYYY in
+// English (slash separator, US month-first order) — verified against real
+// IS Mendelu output for the same record: CZ "14.01.2026" == EN "01/14/2026".
+export function isThisSemester(fulfillmentDate?: string): boolean {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)